massgen/frontend/displays/textual_terminal_display.py

```
@@ -3912,6 +3919,77 @@ def reset_turn_state(self) -> None:
+                )
+                self._refresh_consensus_map()
+
+        def _refresh_consensus_map(self) -> None:
+            """Refresh the compact consensus map from current state."""
+            if not getattr(self, "_consensus_map", None):
+                return
+            try:
+                self._consensus_map.set_state(self._consensus_map_state.snapshot())
+                if self._showing_welcome:
+                    self._consensus_map.add_class("hidden")
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)